### PR TITLE
Again some renaming for metaopt to orion stuff

### PR DESCRIPTION
**Pull request by tsirif**

### CODE REVIEW

File: src/orion/core/resolve_config.py [hunk 1]
```diff
@@ -75,7 +75,7 @@ def is_exe(path):
 # list containing tuples of
 # (environmental variable names, configuration keys, default values)
 ENV_VARS_DB = [
-    ('ORION_DB_NAME', 'name', 'Orion'),
+    ('ORION_DB_NAME', 'name', 'Oríon'),
```
Comment (lamblin) [inline src/orion/core/resolve_config.py]: I'm not ideologically opposed to the accent in strings and so on, on the contrary (we have "Université de Montréal" at a couple places), but the name of the DB itself would be pushing it IMO.

File: docs/src/conf.py [hunk 2]
```diff
@@ -60,8 +60,8 @@
 # General information about the project.
 project = u'orion'
 _full_version = orion.__version__
-author = u'Orion Team - MILA, Université de Montréal'
-copyright = u'2017, Orion Team - MILA, Université de Montréal'
+author = u'Oríon Team - MILA, Université de Montréal'
+copyright = u'2017, Oríon Team - MILA, Université de Montréal'
```
Comment (lamblin) [inline docs/src/conf.py]: 2017--2018?

File: docs/src/conf.py [hunk 3]
```diff
@@ -140,7 +140,7 @@
 # -- Options for HTMLHelp output ------------------------------------------
 
 # Output file base name for HTML help builder.
-htmlhelp_basename = 'Oriondoc'
+htmlhelp_basename = 'Oríondoc'
```
Comment (lamblin) [inline docs/src/conf.py]: Is that part of the URL? If so I would remove the accent.

Comment (bouthilx) [reply docs/src/conf.py]: I agree

**Comment by bouthilx** [comment]
As we already have Universit**é** de Montr**é**al, why not having Or**í**on. I agree with Pascal for DB names and URLs.

**Comment by tsirif** [comment]
Agreed

**Comment by bouthilx** [comment]
@tsirif Quick fix and merge? :)

File: docs/src/conf.py (hunk 2 above)
Comment (tsirif) [reply docs/src/conf.py]: I fixed this in 'feature/docs' #59

File: docs/src/index.rst [hunk 4]
```diff
@@ -1,9 +1,9 @@
-.. Orion documentation master file, created by
+.. Oríon documentation master file, created by
    sphinx-quickstart on Mon Nov 13 19:08:05 2017.
    You can adapt this file completely to your liking, but it should at least
    contain the root `toctree` directive.
 
-Welcome to Orion's documentation!
+Welcome to Oríon's documentation!
 ===================================
```
Comment (lamblin) [inline docs/src/index.rst]: I missed this underline that's too long

**Comment by lamblin** [comment]
Probably still failing because of sphinx 1.7.3

File: docs/src/index.rst (hunk 4 above)
Comment (tsirif) [reply docs/src/index.rst]: Don't worry about it. I am changing it completely in #59

**Comment by bouthilx** [comment]
@tsirif Needs rebase

**Pull request merged**